.github/workflows/bumpVersions.yml

```diff
@@ -6,59 +6,31 @@ on:
     paths:
       - .github/workflows/bumpVersions.yml
       - updatecli.d/**
-    branches:
-      - OPSEXP-**
   workflow_dispatch:
   schedule:
     - cron:  '0 6 * * MON'
 
 jobs:
   updatecli:
-    name: activemq ${{ matrix.activemq_base_version }} version bump
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        activemq_base_version:
-          - 5.16
-          - 5.17
-          - 5.18
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          # Required to trigger the main workflow once this workflow creates a PR
-          token: ${{ secrets.BOT_GITHUB_TOKEN }}
 
       - name: Install Updatecli
         uses: Alfresco/alfresco-build-tools/.github/actions/setup-updatecli@v8.12.0
 
-      - name: Set updatecli required environment variables
-        run: |
-          echo "ACTIVEMQ_BASE_VERSION=${{ matrix.activemq_base_version }}" >> $GITHUB_ENV
-
-      - name: Run updatecli
-        run: |
-          updatecli apply
-
-      - name: Set environment variables after updatecli
-        run: |
-          echo "ACTIVEMQ_VERSION=$(jq -r '.activemq_version' versions/activemq-${{ matrix.activemq_base_version }}.json)" >> $GITHUB_ENV
-
-      - uses: stefanzweifel/git-auto-commit-action@v5
-        id: auto-commit-action
+      - name: Updatecli scm cache
+        uses: actions/cache@v4
         with:
-          branch: bump-activemq-${{ env.ACTIVEMQ_VERSION }}
-          create_branch: true
-          push_options: '--force'
-          commit_user_name: ${{ vars.BOT_GITHUB_USERNAME }}
-          commit_user_email: ${{ vars.BOT_GITHUB_EMAIL }}
+          path: /tmp/updatecli
+          key: updatecli-v1-${{ hashFiles('updatecli.d/**') }}
```

Comment:
Sounds to me like this caching key could be problematic in case an scm is using a floating ref such as master. Don't you think?

Reply:
not a big deal, it's like doing a git pull on a repo which could be a few months behind, still better than cloning from scratch.